lib/OpenQA/Schema/ResultSet/Jobs.pm

```diff
@@ -86,7 +86,7 @@ the return array.
 =cut
 
 sub latest_jobs ($self, $until = undef) {
-    my @jobs = $self->search($until ? {t_created => {'<=' => $until}} : undef, {order_by => ['me.id DESC']});
+    my @jobs = $self->search($until ? {'me.t_created' => {'<=' => $until}} : undef, {order_by => ['me.id DESC']});
```

Comment:
How did you spot this? I didn't see any failing tests. Is it flaky?

Reply:
Oh, I forgot to mention. It was from a logreport email this morning

Reply:
Of course it would be nice to have a failing test in the first place...

Reply:
Seems like we never use `until` anywhere https://github.com/os-autoinst/openQA/pull/5041